src/histolab/tile.py

```diff
@@ -153,14 +153,14 @@ def tissue_ratio(self) -> float:
 
     # ------- implementation helpers -------
 
-    def _has_only_some_tissue(self, near_zero_var_threshold: float = 150.0) -> np.bool_:
+    def _has_only_some_tissue(self, min_var: float = 150.0) -> np.bool_:
         """Check if the tile is composed by only some tissue.
 
         Parameters
         ----------
-        near_zero_var_threshold : float, optional
+        min_var : float, optional
```

Comment:
now the docstring is not correct because it refers to the old filters that were applied